Apps/APN-UIKit/APN UIKit/View/Customisation/CustomDataViewController.swift

```diff
@@ -29,8 +29,23 @@ class CustomDataViewController: BaseViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        addAccessibilityIdentifiersForAppium()
+    }
+    func addAccessibilityIdentifiersForAppium() {
+        if source == .customEvents {
+            setAccessibilityIdentifierTo(eventNameTextField, value: "Event Name Input")
+            setAccessibilityIdentifierTo(propertyNameTextField, value: "Property Name Input")
+            setAccessibilityIdentifierTo(propertyValueTextField, value: "Property Value Input")
+            setAccessibilityIdentifierTo(sendButton, value: "Send Event Button")
+        } else {
+            setAccessibilityIdentifierTo(sendButton, value: "Set \(source == .deviceAttributes ? "Device" : "Profile") Attribute Button")
+            setAccessibilityIdentifierTo(propertyNameTextField, value: "Attribute Name Input")
+            setAccessibilityIdentifierTo(propertyValueTextField, value: "Attribute Value Input")
+        }
```

Comment:
Based on other comment I made with the suggestion of a View extension, this code could maybe be simplified with extensions? 

```swift
propertyNameTextField.setAccessibilityId(source == .customEvents ? "Property Name Input" : "Attribute Name Input")
```

Not a blocker. This is an idea to adopt or ignore.

Reply:
I have added a `UIView` extension to handle this. Thanks for the suggestion.